repo/packages/J/jupyterlab/0/config.json

```diff
@@ -0,0 +1,470 @@
+{
+  "type": "object",
+  "properties": {
+    "service": {
+      "type": "object",
+      "description": "The JupyterLab Notebook Service for Mesosphere DC/OS comes with a variety of pre-installed packages. Further information can be found here https://github.com/dcos/examples/blob/dcos-labs-jupyter-service/jupyter/1.11/README.md",
+      "properties": {
+        "name": {
+          "description": "Name of this service instance.",
+          "type": "string",
+          "default": "/jupyterlab-notebook"
+        },
+        "cmd": {
+          "description": "",
+          "type": "string",
+          "default": "/usr/local/bin/start.sh ${CONDA_DIR}/bin/jupyter lab --notebook-dir=${MESOS_SANDBOX}"
+        },
+        "cpus": {
+          "description": "CPU shares to allocate to each service instance.",
+          "type": "number",
+          "default": 2,
+          "minimum": 1
+        },
+        "force_pull": {
+          "description": "Enable force pull of the image file.",
+          "type": "boolean",
+          "default": false
+        },
+        "gpu_support": {
```

Comment:
There is another field below with gpu configuration. 
Also that field shows different defaults for the added packages e.g., TF 1.8